lib/endpoints/networks.js

```diff
@@ -290,12 +315,28 @@ function getNetwork(req, res, next) {
 
     network = translateNetwork(req, net[0]);
 
+    if (req.sdc.cns && req.account.triton_cns_enabled === 'true') {
+        var account = req.account.uuid;
+        var nets = [network.id];
+
+        req.sdc.cns.getSuffixesForVM(account, nets, function (err, suf) {
+            if (err) {
+                req.log.warn('failed to retrieve DNS suffixes from CNS' +
```

Comment:
Do we fail the whole operation if getSuffixesForVM() fail?  I don't see a callback or anything.

Reply:
Same as above.